internal/pkg/composable/providers/kubernetes/hints.go

```diff
@@ -30,6 +30,8 @@ const (
 	processors  = "processors"
 )
 
+var allSupportedHints = []string{"enabled", "module", integration, datastreams, host, period, timeout, metricspath, username, password, stream, processors}
```

Comment:
why we need here `"enabled", "module"` ? it is not expected to use `module` for the elastic-agent, isn't it?